config/initializers/sidekiq.rb

```diff
@@ -0,0 +1,3 @@
+Sidekiq.configure_server do |config|
+  config.redis = { url: ENV['REDIS_URL'] }
+end
```

Comment:
here we are missing the client configuration
Sidekiq.configure_client do |config|
  config.redis = { url: same url for server }
end